### ISSUE TITLE
Issue with letsencrypt-nginx-proxy-companion

### ISSUE THREAD

**Issue by danil-smirnov** [MattJeanes/jwilder-nginx-proxy-arm64#1]
Hey, thank you for your build, I've just tried to use it but faced the issue with `letsencrypt-nginx-proxy-companion`.

See details in my comment here:
https://github.com/nginx-proxy/nginx-proxy/pull/1470#issuecomment-774425998

**Comment by AmyJeanes** [comment]
Hey, after reading the linked comment and replies it looks like you've got this sorted!

**Comment by danil-smirnov** [comment]
@MattJeanes No, it has been sorted for that branch but your image has the same issue which is not sorted. You can test your image with `letsencrypt-nginx-proxy-companion` and ensure it does not work without `DOCKER_GEN_VERSION` variable defined.

**Comment by danil-smirnov** [comment]
@MattJeanes This is the fix applied by the owner of that branch:

https://github.com/nginx-proxy/nginx-proxy/pull/1470/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R67-R70

I suppose you should apply something similar in your Dockerfile.

**Comment by AmyJeanes** [comment]
Okay I have made this change now and pushed a new docker image up, does this solve the issue?
